msteams-platform/sbs-tab-with-adaptive-cards.yml

```diff
@@ -132,7 +132,7 @@ items:
         ![Bot home page](~/assets/images/adaptive-cards/bot-home-page.png)
 
     1. Open your bot, which is listed in the **Recent resources** section.
-    1. Select **Channels** in the left pane and select **Teams** <img src="~/assets/images/bots/teamsicon.png" alt="Teams icon" width="20"/>.
+    1. Select **Channels** in the left pane and select **Teams** :::image type="content" source="~/assets/images/bots/teamsicon.png" alt-text="Screenshot showing the home page.":::.
```

Comment:
For inline icons this is incorrect usage of Markdown syntax.

Reply:
Markdown rectified.